molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -256,7 +256,7 @@ $.when($,
                     } else {
                         entityCollectionRequest.q.push({
                             operator: 'SEARCH',
-                            value: searchQuery
+                            value: encodeURIComponent(searchQuery)
```

Comment:
Could the encoding be moved to a later point in de code to 'fix' 'all' encoding problems instead of this specific branch ?.

Reply:
The reason why I placed it here, is because the **IF** and **ELSE IF** branches have to do with searching for genomic positions e.g. chromosome_number:position-position 12:19083232-987393292.

All other 'normal' queries are routed to this **ELSE** branch, so technically we do fix all encoding problems for the 'google search' field